packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/FlexibleSyncIntegrationTests.kt

```diff
@@ -64,7 +63,7 @@ class FlexibleSyncIntegrationTests {
 
     @BeforeTest
     fun setup() {
-        app = TestApp(this::class.simpleName, appName = TEST_APP_FLEX, logLevel = LogLevel.ALL)
```

Comment:
It could be helpful to keep it all for CI failures?

Reply:
I don't remember having all the logs available was ever useful on CI. Just having the final exception seems to do the trick?